### PR TITLE
add llama-cpp-python snippet.

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

File: packages/tasks/src/model-libraries-snippets.ts [hunk 1]
```diff
@@ -261,6 +261,24 @@ backbone = keras_nlp.models.Backbone.from_preset("hf://${model.id}")
 `,
 ];
 
+export const llama_cpp_python = (model: ModelData): string[] => [
+	`from llama_cpp import Llama
+llm = Llama.from_pretrained(
+repo_id="${model.id}",
+filename="<QUANT FILE NAME>",
```
Comment (Vaibhavs10) [inline packages/tasks/src/model-libraries-snippets.ts:268]: Not sure what's the best way to deal with quant file name here?

File: packages/tasks/src/model-libraries.ts [hunk 2]
```diff
@@ -314,6 +314,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"liveportrait/landmark.onnx"`,
 	},
+	"llama-cpp-python": {
+		prettyLabel: "llama-cpp-python",
+		repoName: "llama-cpp-python",
+		repoUrl: "https://github.com/abetlen/llama-cpp-python",
+		snippets: snippets.llama_cpp_python,
+		filter: true,
```
Comment (julien-c) [inline packages/tasks/src/model-libraries.ts:322]:
```suggestion
```

Comment (julien-c) [reply packages/tasks/src/model-libraries.ts:322]: models won't be tagged explicitly as llama-cpp-python through https://github.com/huggingface-internal/moon-landing/pull/10885, only implicitly

File: packages/tasks/src/model-libraries-snippets.ts [hunk 3]
```diff
@@ -261,6 +261,24 @@ backbone = keras_nlp.models.Backbone.from_preset("hf://${model.id}")
...
+llm.create_chat_completion(
+		messages = [
+			{
+				"role": "user",
+				"content": "What is the capital of France?"
+			}
+		]
+)
+	
+`,
```
Comment (julien-c) [inline packages/tasks/src/model-libraries-snippets.ts:279]:
```suggestion
	`from llama_cpp import Llama

llm = Llama.from_pretrained(
	repo_id="${model.id}",
	filename="{{GGUF_FILE}}",
)

llm.create_chat_completion(
		messages = [
			{
				"role": "user",
				"content": "What is the capital of France?"
			}
		]
)
```

**Review by julien-c — APPROVED** [review]
You can use `{{GGUF_FILE}}` to display a selector like for llama.cpp, i've opened a PR to support this in our internal codebase

File: packages/tasks/src/model-libraries.ts (hunk 2 above)
Comment (julien-c) [reply packages/tasks/src/model-libraries.ts:322]: let's remove it ok?

**Review by julien-c — APPROVED** [review]
let's merge to get the internal PR to be green:)

**Pull request merged**